doc/awx-setup.md

```diff
@@ -0,0 +1,46 @@
+# Recreating the AWX server
+
+First step is to run the AWX playbook:
+
+```sh 
+$ ansible-playbook playbooks/create-awx.yml
+```
+
+_note_: the playbook is currently targeted against `infra-ibm-ubuntu2004-x64-1` so if that changes don't forget to update the playbook
```

Comment:
Maybe we could target against the host with the `ansible` alias set?

Reply:
doesnt seem to be possible

Reply:
> Maybe we could target against the host with the `ansible` alias set?

Hmm rereading the ansible docs this might not be straightforward. Let's stick to how the PR is for now.